Libraries/test/TestServerlessApp.IntegrationTests/DeploymentScript.ps1

```diff
@@ -0,0 +1,38 @@
+$ErrorActionPreference = 'Stop'
+try
+{
+    Push-Location $PSScriptRoot
+    $guid = New-Guid
+    $suffix = $guid.ToString().Split('-') | Select-Object -First 1
+    $identifier = "test-serverless-app-" + $suffix
+    cd ..\TestServerlessApp
+
+    # Replace bucket name in aws-lambda-tools-defaults.json
+    $line = Get-Content .\aws-lambda-tools-defaults.json | Select-String s3-bucket | Select-Object -ExpandProperty Line
+    $content = Get-Content .\aws-lambda-tools-defaults.json
+    $content | ForEach-Object {$_ -replace $line, "`"s3-bucket`" : `"$identifier`","} | Set-Content .\aws-lambda-tools-defaults.json
+
+    # Replace stack name in aws-lambda-tools-defaults.json
+    $line = Get-Content .\aws-lambda-tools-defaults.json | Select-String stack-name | Select-Object -ExpandProperty Line
```

Comment:
Just pass in the stack name when calling `deploy lambda deploy-serverless` instead of changing the aws-lambda-defaults.json file.

Reply:
Refer to the above comment